vmb/matrimony/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+from . import views
+
+urlpatterns = [
+    path("response/<int:id>", views.response, name="response"),
+    path("match/<int:id>", views.match, name="match"),
```

Comment:
As a shortcut for now, you can just handle `post` request in this URL to update response. This will avoid using AJAX for now.